samples/net/mqtt_azure/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _mqtt-azure-sample:
+
+MQTT Azure Sample
+#################
+
+Overview
+********
+
+This sample application demonstrates how a MQTT client
+can publish messges to Azure Cloud IoT hub based on MQTT protocol.
```

Comment:
change to: ``messages to an Azure`` (fix misspelling and add an article)

Reply:
@dbkinder thanks for review. I will fix it and update the PR.